bigquery_etl/glean_usage/baseline_clients_daily.py

```diff
@@ -37,12 +36,14 @@
     help="Also write the query text underneath the given sql dir",
 )
 parser.add_argument(
-    "--views_only",
-    "--views-only",
+    "--output_only",
+    "--output-only",
+    "--views_only",  # Deprecated name
```

Comment:
`--output-only` and `--views-only` are aliases?

Reply:
`views_only` was implemented to allow Jenkins to stably generate the view files for views publishing. But for the generated-sql context, we want to output the generated _queries_ as well. The views publishing use case isn't hurt by having the queries generated as well. I want to leave `views_only` in place since Jenkins currently hits that flag, but it's no longer semantically correct.

We can clean this up later here if we update the Jenkins job.